apps/web/app/(auth-pages)/sign-in/page.tsx

```
@@ -1,15 +1,25 @@
 'use client'
 
+import { AuthError } from '@supabase/supabase-js'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)